data/low_income_thresholds.json

```diff
@@ -185,6 +185,656 @@
         "7": 220050,
         "8": 234300
       }
+    },
+    "co-xcel-energy-ev": {
+      "type": "county-hhsize",
+      "incentives": [
+        "CO-567",
+        "CO-568"
+      ],
+      "source_url": "https://ev.xcelenergy.com/ev-rebate-co",
+      "thresholds": {
+        "08001": {
```

Comment:
I need help understanding how this works, and I don't think we have this pattern elsewhere with `thresholds`... can we do a quick huddle in Slack on it tomorrow?

Reply:
Sure we can talk about it. There's some other examples if you do cmd-f in the file for "county-hhsize".

Reply:
sharing for anyone who sees this later: Cameron showed me the `type` field here that I overlooked, and [this function ](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/src/lib/low-income.ts#L14) that checks the type